src/lib/AnnotationModule.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    bdlBoxBlue,
+    bdlGreenLight,
+    bdlWatermelonRed,
+    bdlYellorange,
+    bdlYellow,
+    bdlGrimace,
+} from 'box-ui-elements/es/styles/variables';
+import Cache from './Cache';
+
+export const ANNOTATION_COLOR_KEY = 'annotation-color';
+
+export const AnnotationColor = {
+    BOX_BLUE: bdlBoxBlue,
+    GREEN_LIGHT: bdlGreenLight,
+    WATERMELON_RED: bdlWatermelonRed,
+    YELLORANGE: bdlYellorange,
+    YELLOW: bdlYellow,
+    GRIMACE: bdlGrimace,
+};
+
+export default class AnnotationModule {
+    private cache: Cache;
+
+    constructor({ cache }: { cache: Cache }) {
+        this.cache = cache;
+
+        if (this.cache && !this.cache.get(ANNOTATION_COLOR_KEY)) {
```

Comment:
Can `this.cache` be undefined? We don't have this check elsewhere. Can this call `this.getColor()`, instead?

Reply:
We can use this.getColor and I've removed the existence check / updated tests for all classes that inherit from BaseViewer.